fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/properties/FDBRecordContextProperties.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * FDBRecordContextProperties.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.provider.foundationdb.properties;
+
+import com.apple.foundationdb.annotation.API;
+import com.apple.foundationdb.record.RecordCoreException;
+import com.apple.foundationdb.record.logging.LogMessageKeys;
+import com.google.common.collect.ImmutableMap;
+
+import javax.annotation.Nonnull;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Adopter uses this wrapper of mapping of {@link FDBPropertyKey} to configure parameters for a {@link com.apple.foundationdb.record.provider.foundationdb.FDBRecordContext}.
+ * The instances of {@link FDBPropertyKey} and how they are used are defined in Record Layer.
+ */
+@API(API.Status.EXPERIMENTAL)
+public class FDBRecordContextProperties {
```

Comment:
Regardless of whether it retains the `FDB` prefix, this might be clearer as `PropertyStorage` instead of just `Properties`. In particular, when I see the `Properties` suffix, I expect it to contain a much of property definitions (like `LuceneRecordContextProperties` does)

Reply:
This makes sense. Fixed it.